doc/contributing.rst

```diff
@@ -159,8 +182,13 @@ use the command::
 
     $ make html-noplot
 
+Finally, to view the documentation, do::
+
+    $ make view
```

Comment:
I never knew this before! This is easier than calling `$firefox _build/html/index.html`.